### ISSUE TITLE
some sprites not layering on background properly

### ISSUE THREAD

**Issue by davidjtferguson** [davidjtferguson/silly-sam#77]
Ever since removing the background tiles in tiled and letting the sprites just lay on the screen cleared colour the boarders to some sprites (clouds, tree tufts) have had a weird dark outline. Weirldy, trees don't have this (or at least it's not visible)

Could re-instate background layer, would be better to figure it out and fix it instead of having to draw extra stuff.

**Comment by davidjtferguson** [comment]
I just added PR #86 which allows sam to go behind objects. I noticed when doing so that the weird halo draws behind sam but the objects draw infront of sam. So it seems like some weird background version of the object is being drawn behind it for some reason (as opposed to some transparancy issue like I've previously theorised).
So need to figure out how to not draw this weird shadow object. It's not textured, so seems kinda similar to the issue I had before replacing the objects/toys draw function where a shadow version was being drawn of those. So that leads me to `Map:drawObjectLayer(layer)` in STI's init.lua to see if anything there draws behind for some reason.
